dbt/adapters/bigquery/clients.py

```diff
@@ -1,8 +1,7 @@
 from google.api_core.client_info import ClientInfo
 from google.api_core.client_options import ClientOptions
-from google.api_core.retry import Retry
 from google.auth.exceptions import DefaultCredentialsError
-from google.cloud.bigquery import Client as BigQueryClient
+from google.cloud.bigquery import Client as BigQueryClient, DEFAULT_RETRY
```

Comment:
As it turns out, `Retry()` with no arguments doesn't really have defaults that we want, but this is what we were using for the BQ client factory. We weren't retrying at all on other factories.